src/Sylius/Bundle/ShopBundle/templates/shared/account/address_book/form/fields.html.twig

```diff
@@ -0,0 +1,3 @@
+<div class="mb-4">
+    {% hook 'address' with { _prefixes: ['sylius_shop.shared.form'] } %}
```

Comment:
Are the custom (for checkout and address book) hooks also triggered here or somewhere? 🤔

Reply:
No they were not, but I fixed it and now it works properly